release.py

```diff
@@ -0,0 +1,243 @@
+#!/usr/bin/env python3
```

Comment:
Maybe put this in the `scripts/` directory to keep the top-level a little tidier